### ISSUE TITLE
multi-platform push is broken

### ISSUE THREAD

**Issue by bruce-szalwinski-he** [vagmcs/poetry-docker-plugin#2]
Based on the docs, configuring a multi-platform build and push is done using:

```
poetry docker --platform linux/amd64 --platform linux/arm64 --push
```

While the build part works, the push does not.

- build [joins all of the platforms](https://github.com/vagmcs/poetry-docker-plugin/blob/master/poetry_docker_plugin/docker_builder.py#L331)
- push fails to [join all of the platforms](https://github.com/vagmcs/poetry-docker-plugin/blob/master/poetry_docker_plugin/docker_builder.py#L353)

fails with:

```
ERROR: "['linux" is an invalid OS component of "['linux/amd64'": OSAndVersion specifier component must match "^([A-Za-z0-9_-]+)(?:\\(([A-Za-z0-9_.-]*)\\))?$": invalid argument
```

**Comment by vagmcs** [comment]
Hi @bruce-szalwinski-he,

Thank you for reporting and fixing the issue. I will have a look at #3 tomorrow and publish a new version.

**Comment by vagmcs** [comment]
Released version 0.5.2. I am closing the issue. Thank you for your contributions.
